src/features/notification/types.ts

```diff
@@ -115,6 +116,7 @@ export interface NotificationContent {
   addedRoles?: string
   type?: string | number
   newUrl?: string
+  Roles?: string[]
```

Comment:
why upper case? this is not our coding style.

Reply:
That is how we are getting the info from server.